README.md

```diff
@@ -53,6 +54,12 @@ The version of Tutor where you want to run the integration tests. You can specif
 **Optional**  
 The python version to run the integration tests with. Make sure to use a version fully supported by the plugin.
 
+### `tutor_plugins`
+
+**Optional**  
+The list of Tutor plugins to install. Provide them as a space-separated string.  
```

Comment:
```suggestion
The list of Tutor Index plugins to install. Only plugins available in the official Tutor plugin index can be installed using this option. Provide them as a space-separated string.
```

we could also link the documentation and just say something like:
The list of Tutor Index plugins to install. Refer to the official tutor documentation https://docs.tutor.edly.io/reference/cli/plugins.html#tutor-plugins-install ...

or we could link the documentation and add this explanation in the overview section https://github.com/eduNEXT/integration-test-in-tutor/pull/8/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R107-R108

Reply:
Done: https://github.com/eduNEXT/integration-test-in-tutor/commit/fb3bb444c5c9e17a6a337fb470c7adf93c97dc53